presign/control/views/participant.py

```diff
@@ -149,9 +149,14 @@ def post(self, *args, **kwargs):
             raise Http404()
         success_msg = STATE_CHANGE_STRINGS[action]["success_msg"]
 
+        # if we have content in custom-message form parameter, we need to save it
+        custom_message = self.request.POST.get("custom-message")
```

Comment:
Thats what forms are for

Reply:
Rebuild with forms, now needs a better design and more testing.